### ISSUE TITLE
BUG : Import CURL body that contains single-quote

### ISSUE THREAD

**Issue by trayburn** [a85/POSTMan-Chrome-Extension#1999]
There is a bug in the **Import** feature of Postman that fails on valid data parameters which contain a single-quote inside of double-quotes:

```
curl -X POST -H 'Content-Type: application/json' -d '{
    "Handle": "this isn\'t \" a test"
}
' "http://test.org"
```

This fails to import, and that does not change if you instead make it `"this isn'\''t \" a test"` for the JSON value.  Removing the single quote in the middle altogether and making it `"this isnt \" a test"` allows it to proceed correctly

**Comment by abhijitkane** [comment]
@trayburn 
This: `curl -X POST -H 'Content-Type: application/json' -d '{"Handle": "this isn\'t a test"}' http://httpbin.org/post` won't work in the shell either. 

You'll need to use the unicode escape sequence - `curl -X POST -H 'Content-Type: application/json' -d '{"Handle": "this isn\u0027t a test"}' http://httpbin.org/post`.

**Comment by greglockwood** [comment]
A related problem I am finding is that importing CURL commands that have the data parameter surrounded by single quotes ends up with the single quotes in the body text inside Postman, rather than treating them correctly as delimeters and omitting them from the body text.

Postman version 4.4.1
win / x86-32
Chrome 51.0.2704.103

**Comment by abhijitkane** [comment]
@greglockwood Can you give an example of a curl command that fails?

```
curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d '{
    "hello": "world"
}' "http://google.com"
```

works correctly.

**Comment by abhijitkane** [comment]
@greglockwood @trayburn This has been fixed in v4.4.2 of the app (Chrome only). Can you update (https://www.getpostman.com/docs/updating) and check?
